손슬기/week1/2023.03.02./boj10872/boj10872.py

```diff
@@ -0,0 +1,8 @@
+n = int(input())
+if n == 0:
```

Comment:
if - else 가 없어도 괜찮을 것 같습니다!
```suggestion
f = 1
for i in range(1, n + 1):
    f*=i
print(f)
```